bin/sync/article_cleaner.rb

```diff
@@ -0,0 +1,35 @@
+require 'yaml'
+require 'fileutils'
+
+class ArticleCleaner
+  BLOG_DIR = 'content/blog'.freeze
+  YAML_FILE = 'sync_status.yml'.freeze
+  ARTICLE_FILE = 'index.md'.freeze
+
+  def initialize
+    @slugs = load_slugs_from_yaml
+  end
+
+  def call
```

Comment:
Please rename to `perform` just for the following https://github.com/jetthoughts/how-we-work